commenting out the async-signal-unsafe code from the
sshsigdie() function

--- a/log.c	2024-06-28 19:27:20.350862974 +0000
+++ b/log.c	2024-06-28 19:28:40.833530494 +0000
@@ -451,12 +451,14 @@ void
 sshsigdie(const char *file, const char *func, int line, int showfunc,
     LogLevel level, const char *suffix, const char *fmt, ...)
 {
+#if 0
 	va_list args;
 
 	va_start(args, fmt);
 	sshlogv(file, func, line, showfunc, SYSLOG_LEVEL_FATAL,
 	    suffix, fmt, args);
 	va_end(args);
+#endif
 	_exit(1);
 }
 
